libs/ui/src/precinct_scanner_tally_qrcode.tsx

```diff
@@ -1,82 +1,69 @@
-import { CompressedTally, ElectionDefinition } from '@votingworks/types';
+import {
+  CompressedTally,
+  Election,
+  ElectionDefinition,
+} from '@votingworks/types';
 import { format, formatFullDateTimeZone } from '@votingworks/utils';
 import { DateTime } from 'luxon';
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import styled from 'styled-components';
 import { LogoMark } from './logo_mark';
 import { Prose } from './prose';
 import { ReportSection } from './tally_report';
 import { Text } from './text';
 import { QrCode } from './qrcode';
 
-const QrCodeWrapper = styled.div`
-  width: 25%;
-`;
-
-interface Props {
-  pollsToggledTime: number;
+interface SignedQuickResultsReportingUrlProps {
   electionDefinition: ElectionDefinition;
   signingMachineId: string;
-  isPollsOpen: boolean;
   isLiveMode: boolean;
   compressedTally: CompressedTally;
 }
 
-export function PrecinctScannerTallyQrCode({
-  pollsToggledTime,
+export async function getSignedQuickResultsReportingUrl({
   electionDefinition,
   signingMachineId,
-  isPollsOpen,
   isLiveMode,
   compressedTally,
-}: Props): JSX.Element {
-  const { election, electionHash } = electionDefinition;
-  const [resultsReportingUrl, setResultsReportingUrl] = useState('');
-  const pollsAction = isPollsOpen ? 'Opened' : 'Closed';
-  const electionDate = format.localeWeekdayAndDate(new Date(election.date));
-
-  useEffect(() => {
-    let isCurrentRender = true;
```

Comment:
There used to be logic here for resigning on re-render, but if we don't intend to re-render this component (and we don't anywhere in any app after this PR) then we don't need it.